Source/RapyutaSimulationPlugins/Private/Tools/RRROS2SkeletalMeshStatePublisher.cpp

```diff
@@ -35,7 +35,6 @@ void URRROS2SkeletalMeshStatePublisher::SetTargetRobot(ARobotVehicle* InRobot)
         SkeletalMeshComp = skeletalMeshComponents[0];
         const auto bonesNum = SkeletalMeshComp->GetBoneSpaceTransforms().Num();
         UE_LOG_WITH_INFO(LogRapyutaCore, Log, TEXT("[%s] has %d bones"), *SkeletalMeshComp->GetName(), bonesNum);
-        check(bonesNum > 0);

```

Comment:
error log for (bonesNum == 0)

Reply:
I don't think bonesNum==0 don't cause error

Reply:
Yeap, a skeletal mesh component is definitely invalid upon no bones, but more essentially, it seems this class is unused?

Reply:
yes this class is not used for now.